reports/proposal.Rmd

```diff
@@ -30,15 +30,11 @@ Being the largest city in British Columbia, Vancouver is an attractive city to c
 
 # Research Questions and Usage Scenarios
 
+Mary is a new to Vancouver, who is not very familiar with the neighborhoods. She wants to settle down by renting a place in Vancouver, and now she faces the problem of which neighborhoods to choose. One important factor she takes into account is the crime incidents. She wants to compare the safety of neighborhoods in the aspect of least crime incidents. Moreover, since certain types of crimes (*e.g.* homicide) are more serious and life-threatening than others, she is also willing to know the types of crimes that occur most frequently in a particular neighborhood. In case that she only wants to accommodate for a short period, it would be helpful to know the specific crime distribution during that time frame.
```

Comment:
Let's take a look at Mary. She recently moved to Vancouver but is not very familiar with the neighborhoods.